src/main.cpp

```diff
@@ -2784,6 +2786,42 @@ void static UpdateTip(CBlockIndex *pindexNew) {
       Checkpoints::GuessVerificationProgress(chainParams.Checkpoints(), chainActive.Tip()), pcoinsTip->DynamicMemoryUsage() * (1.0 / (1<<20)), pcoinsTip->GetCacheSize());
 
     cvBlockChange.notify_all();
+    
+    
+
+    // MVHF-BU
+    const int MVHF_FORK_BLOCK = 50000;
+
+    //// 	Auto Backup At Block 		////
+    // Test autobackupwalletpath argument
+    if (!fAutoBackupDone)
+    {
+    	std::string strWalletBackupFile = GetArg("-autobackupwalletpath", "");
+
+		int BackupBlock = GetArg("-autobackupblock", MVHF_FORK_BLOCK - 1);
+
+		//LogPrintf("DEBUG: autobackupwalletpath=%s\n",strWalletBackupFile);
+		//LogPrintf("DEBUG: autobackupblock=%d\n",BackupBlock);
+
+		if(strWalletBackupFile != "") {
+
+			if (GetBoolArg("-disablewallet", false)) {
+				LogPrintf("-disablewallet and -autobackupwalletpath conflict so automatic backup disabled.");
```

Comment:
outputting here is OK, but the conflict should be detected and logged at startup already.
See InitParameterInteraction() in src/init.cpp. Also, the new parameters should be included in HelpMessage() there.
Maybe there is some more documentation as well, I'm not sure but it should be checked.